libsplinter/src/biome/migrations/error.rs

```diff
@@ -16,19 +16,19 @@ use std::error::Error;
 use std::fmt;
 
 #[derive(Debug)]
-pub struct ConnectionError {
+pub struct MigrationError {
```

Comment:
This is part of the public stable API.  You could create a type alias for the old name and deprecate it.

Reply:
Updated